frontend/src/App.css

```
@@ -44,3 +44,8 @@
 .read-the-docs {
   color: #888;
 }
+
+html.dark svg,
+.svg-lightbox {
+  filter: invert(1) hue-rotate(180deg) brightness(1);
```

100% is the default for invert, and `brightness(1)` has no effect.

```suggestion
  filter: invert() hue-rotate(180deg);
```